src/context/ColonyCreateModalContext/ColonyCreateModalContextProvider.tsx

```diff
@@ -10,9 +10,19 @@ const ColonyCreatedModalContextProvider = ({
   const [isColonyCreatedModalOpen, setIsColonyCreatedModalOpen] =
     useState<boolean>(false);
 
+  useEffect(() => {
+    const handleCloseModals = () => {
+      setIsColonyCreatedModalOpen(false);
+    };
+
+    window.addEventListener('closeModals', handleCloseModals);
+
+    return () => window.removeEventListener('closeModals', handleCloseModals);
+  }, []);
```

Comment:
Is there a use case for closing the modals like so yet? I couldn't trigger a tour while any of the modals was open. I'm wondering if it would be better to only add this once needed.

Also since that similar block of code is repeated across a few components, a shared hook sounds like a good idea. We should also use an enum for the event name.

Reply:
Which modals did you test? I implemented it due to the issue occurring when opening the tour while the token locking modals were open, I have just tested again and is needed. I agree though, this would be better as a shared hook and I will use an enum.

Reply:
You're right, I must've missed that one